docs/usage.md

```diff
@@ -377,13 +377,15 @@ already contains everything you need.
 Containerfile.baseimage:
 
 ```Dockerfile
-FROM quay.io/centos/centos:stream8
-
-# python3.8 runtime, C build dependencies
-RUN dnf -y install \
-        python38 \
-        python38-pip \
-        python38-devel \
+FROM quay.io/centos/centos:stream9
+
+# python3.9 runtime, C build dependencies
+RUN dnf -y install 'dnf-command(config-manager)' && \
+    dnf config-manager --set-enabled crb && \
```

Comment:
Disclaimer: I didn't know we had an example like this until you pointed it out.

CS is a rolling distro which breaks package updates more often than not sadly (at least it used to be the case). I mean what you have here is all fine but strictly from docs POV I just wonder if it deserves the promotion (even just as an example) since it requires the Content Ready Builder repo to enable headers. Anyway, I think this hunk would be slightly simplified if you replaced CS with an example using AlmaLinux or Rocky where you don't need either the config manager or the CRB repo.

Reply:
Atomic reactor [uses](https://github.com/containerbuildsystem/atomic-reactor/blob/master/Dockerfile#L1) Fedora, so we might as well just do the same.

Reply:
Works too.